docs/running_lbann.rst

```diff
@@ -1,96 +1,434 @@
 .. role:: bash(code)
           :language: bash
+.. role:: python(code)
+          :language: python
 
-====================
+============================================================
 Running LBANN
-====================
+============================================================
 
-The basic template for running LBANN is
+------------------------------------------------
+Anatomy of an LBANN experiment
+------------------------------------------------
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Parallelism
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+LBANN is run under the `MPI
```

Comment:
Pedantic quibble: MPI is a standard. MPSD/MPMD and message passing are paradigms.